modules/sync/sync-tax.php

```diff
@@ -105,7 +105,7 @@ protected function maybe_translate_terms( $object_id, $terms, $taxonomy, $lang )
 				 * @param int    $term    Source term id
 				 * @param string $lang    Language slug
 				 */
-				if ( $term_id = apply_filters( 'pll_maybe_translate_term', $this->model->term->get_translation( $term, $lang ), $term, $lang ) ) {
+				if ( $term_id = apply_filters( 'pll_maybe_translate_term', (int) $this->model->term->get_translation( $term, $lang ), $term, $lang ) ) {
```

Comment:
What's the reason for this change? Did you check that we never explicitely test `false` when using the filter?

Reply:
I found only `PLL_Sync_Content::duplicate_term()` that uses this filter and [it uses `empty()`](https://github.com/polylang/polylang-pro/blob/3567e9416947690bdb6d882df0a02d6b118a3c8b/modules/sync/sync-content.php#L187).